### PR TITLE
TOOL-30 Updated to check for gems.locked too.

### PR DESCRIPTION
**Pull request by richard-bogdan-bitrise**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -345,13 +345,12 @@ func main() {
 	log.Printf("Searching for Gemfile.lock with cocoapods gem")
 
 	// Check Gemfile.lock for CocoaPods version
-	gemfileLockPth := filepath.Join(podfileDir, "Gemfile.lock")
-	isGemfileLockExists, err := pathutil.IsPathExists(gemfileLockPth)
-	if err != nil {
+	gemfileLockPth, err := gems.GemFileLockPth(podfileDir)
+	if err != nil && err != gems.ErrGemLockNotFound {
```
Comment (lpusok) [inline main.go]: PLease update the error message "Failed to check Gemfile.lock at..." to not specifically refer Gemfile.lock, remove gemfileLockPth (as is empty string propably) or print the list of known gemfile names.

File: main.go [hunk 2]
```diff
@@ -345,13 +345,12 @@ func main() {
...
 		failf("Failed to check Gemfile.lock at: %s, error: %s", gemfileLockPth, err)
 	}
 
-	if isGemfileLockExists {
+	if gemfileLockPth != "" {
 		// CocoaPods exist search for version in Gemfile.lock
 		log.Printf("Found Gemfile.lock: %s", gemfileLockPth)
```
Comment (lpusok) [inline main.go]: Could update error messages to "Found gems lockfile..."

File: main.go (hunk 1 above)
Comment (richard-bogdan-bitrise) [reply main.go]: Updated code.

File: main.go (hunk 2 above)
Comment (richard-bogdan-bitrise) [reply main.go]: Updated.

**Comment by richard-bogdan-bitrise** [comment]
@lpusok: had to rebase and resolve conflicts, as other PR was merged.

**Pull request merged**